site/source/docs/porting/exceptions.rst

```diff
@@ -32,19 +23,54 @@ For example, build the following program with ``emcc -fexceptions``:
       return 0;
     }
 
-Running it will print ``throw...`` and then ``catch!``.
+The first ``throw`` will abort the program and you'll see something like this in the output:
+
+.. code-block:: text
+
+  throw...  
+  exception thrown: 5246024 - Exception catching is disabled, this exception cannot be caught. Compile with -s NO_DISABLE_EXCEPTION_CATCHING or -s EXCEPTION_CATCHING_ALLOWED=[..] to catch.
```

Comment:
I think the flag is `DISABLE_EXCEPTION_CATCHING=0`

Reply:
oh wait, that is what the error actually says... should we fix that?

Reply:
I... guess? Are they not interchangeable? I haven't tried it, but if the error message is wrong and that option doesn't work, then yeah, absolutely let's fix it.

Reply:
🤯